crates/bevy_core_pipeline/src/oit/oit_draw.wgsl

```diff
@@ -5,6 +5,12 @@
 #ifdef OIT_ENABLED
 // Add the fragment to the oit buffer
 fn oit_draw(position: vec4f, color: vec4f) {
+    // Don't add fully transparent fragments to the list
+    // because we don't want to have to sort them in the resolve pass
+    // TODO should this be comparing with < espilon ?
```

Comment:
I suppose this could be something like:
```rust
const EPSILON: f32 = 4.88e-04;
if color.a < EPSILON { ... }
```
But tbh I'm not seeing that this is done when comparing color values anywhere in the codebase.

Reply:
The idea behind that comment is more about having a cutoff for almost transparent pixels. Instead of fully transparent, the reason is that every fragment we add to the list costs us more memory. Right now it doesn't really matter because we have a fixed buffer size, but eventually I want to work on making it smaller.